internal/controller/policyserver_controller_deployment.go

```diff
@@ -343,6 +344,34 @@ func (r *PolicyServerReconciler) adaptDeploymentSettingsForPolicyServer(policySe
 			},
 		)
 	}
+
+	if r.ClientCAConfigMapName != "" {
+		policyServerDeployment.Spec.Template.Spec.Volumes = append(
+			policyServerDeployment.Spec.Template.Spec.Volumes,
+			corev1.Volume{
+				Name: clientCAVolumeName,
+				VolumeSource: corev1.VolumeSource{
+					ConfigMap: &corev1.ConfigMapVolumeSource{
+						LocalObjectReference: corev1.LocalObjectReference{
+							Name: r.ClientCAConfigMapName,
+						},
+						Items: []corev1.KeyToPath{
+							{
+								Key:  constants.ClientCACert,
+								Path: constants.ClientCACert,
+							},
+						},
+					},
+				},
+			},
+		)
+
+		admissionContainer := &policyServerDeployment.Spec.Template.Spec.Containers[0]
+		admissionContainer.Env = append(admissionContainer.Env, corev1.EnvVar{
+			Name:  "KUBEWARDEN_CLIENT_CA_FILE",
+			Value: constants.ClientCACert,
+		})
```

Comment:
Here we might have a problem. We have to inject two CAs: the one used by the Kubernetes API server and the one used by `audit-scanner`. Inside of policy-server we're planning to allow the same cli flag to be repeated multiple times. I don't think there's a way to achieve the same thing (provide multiple values) using the environment variable.

Maybe we have to change the deployment to use cli flags for that.

Reply:
We could parse both the env variable and the cli flag with  the [value_delimiter](https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_delimiter) set.